UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/FileShareService/Response/FulfilmentDataResponse.cs

```diff
@@ -15,5 +15,7 @@ public class FulfilmentDataResponse
         public IEnumerable<string> FileUri { get; set; }
 
         public IEnumerable<BatchFile> Files { get; set; }
+        
+        public int TotalHitCountForQueryFileShareService { get; set; }
```

Comment:
Kindly change variable name from TotalHitCountForQueryFileShareService to FileShareServiceSearchQueryCount to maintain consistency

Reply:
Done